raiden_contracts/deploy/__main__.py

```diff
@@ -357,11 +359,26 @@ def verify(ctx, rpc_provider, contracts_version):
     verify_deployed_contracts(web3, contract_manager)
 
 
+class Contract(TypedDict):
+    address: Address
+    transaction_hash: str
+    block_number: int
+    gas_cost: int
+    constructor_arguments: Any
+
+
+class DeployedContracts(TypedDict):
+    chain_id: int
+    contracts: Dict[str, Contract]
+    contracts_version: str
+
+
```

Comment:
Ah, one comment here though - can we move this at the beginning off the file or at the end?

Reply:
Will do.

Reply:
And maybe rename the `Contract` class to something more specific, to be easily grep-able.